testdata/Makefile

```
@@ -0,0 +1,97 @@
+CC = clang
+CXX = clang++
+
+# Keep this flags in sync with the flags used in the make integration
+# see also `internal/cmd/run/run.go`
+CFLAGS = -fsanitize=fuzzer,address,undefined \
+		-fno-sanitize=function,vptr \
+		-fsanitize-recover=address \
+		-fsanitize-address-use-after-scope \
+		-g
+
+CXXFLAGS = -fsanitize=fuzzer,address,undefined \
+		-fno-sanitize=function,vptr \
+		-fsanitize-recover=address \
+		-fsanitize-address-use-after-scope \
+		-std=c++17 \
+		-g
+
+BUILD_DIR = build
+SHARED_DIR = shared
+FUZZ_TARGET_DIR = fuzz-targets
+# get all c/cpp files and remove file extension
+EXT :=
+OUT_OPTION := -o 
+INCLUDE_OPTION := -I 
+NOLINK_OPTION := -c
+OUT_OBJ_OPTION := -o
+DISABLE_UBSAN := 0
+
+# handle MSVC specific options
+ifeq ($(OS),Windows_NT)
+	CC = cl
+	CXX = cl
+	CFLAGS = -fsanitize=fuzzer \
+	  -fsanitize=address \
```

I would suggest using the MSVC flag style consistently, even if there is no functional difference.
```suggestion
	CFLAGS = /fsanitize=fuzzer \
	  /fsanitize=address \
```

---

done